test/JDBC/sql_expected/BABEL-IDENTITY.out

```diff
@@ -423,24 +423,115 @@ int#!#int
 ~~END~~
 
 
--- Test get id max/min helper functions
-SELECT sys.get_min_id_from_table(';drop table master_dbo.test_table1;', 'master_dbo', 'test_table1');
-GO
+-- Test that identity counters shouldn't rolled back even if the transaction
+-- is rolled back.
+CREATE TABLE dbo.t1_identity_1(a int identity primary key, b int);
+SET IDENTITY_INSERT dbo.t1_identity_1 ON;
+INSERT INTO dbo.t1_identity_1 (a,b) VALUES (1,1);
+go
+~~ROW COUNT: 1~~
+
+
+-- Check transaction rollback should increase/decrease identity
+BEGIN TRAN t1; INSERT INTO dbo.t1_identity_1 (a,b) VALUES (300,2); ROLLBACK TRAN t1;
+SELECT @@IDENTITY; SELECT IDENT_CURRENT('dbo.t1_identity_1'); SELECT SCOPE_IDENTITY();
```

Comment:
Can we have an upgrade test for this? Even after mVU/MVU, do we expect the same value, e.g., 300?

Reply:
For IDENT_CURRENT(), we expect the same value after mVU/MVU. It fetches the last used value for the sequence from the PG catalog. 
Rest of the two functions are session specific. We already have existing upgrade tests for IDENT_CURRENT(). Do you think we need to test some other scenario?
One scenario could be all previous releases rollbacks the sequence value when transaction rollbacks, whereas after an upgrade to this version, the value is not rolled back.

Reply:
Thanks for checking. In that case, I don't think we need further upgrade tests.